src/Evolution/Systems/ScalarAdvection/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ spectre_target_sources(
   Characteristics.cpp
```

Comment:
can you reword the commit message to `Add ScalarAdvection velocity field function` so there's some context